arch/aarch64/mm/page.c

```diff
@@ -258,6 +265,10 @@ int page_fault_handler(size_t viraddr)
 default_handler:
 	spinlock_irqsave_unlock(&page_lock);
 
+	/* indicate unrecoverable page fault to the hypervisor */
+	uhyve_pfault_t arg = {pc, viraddr, -1};
+	uhyve_send(UHYVE_PORT_PFAULT, (unsigned)virt_to_phys((size_t)&arg));
```

Comment:
Does this really make sense? Copy the (physical) address of a stack variable to uhyve? Is there any guarantee that nobody will reuse the stack before the data at this address is read?